plugins/janus_streaming.c

```diff
@@ -79,11 +79,14 @@ videobufferkf = yes|no (whether the plugin should store the latest
 	keyframe and send it immediately for new viewers, EXPERIMENTAL)
 databuffermsg = yes|no (whether the plugin should store the latest
 	message and send it immediately for new viewers)
+x-audio-multicast-iface = network interface IP address or device name to listen on when receiving multicast audio
+x-video-multicast-iface = network interface IP address or device name to listen on when receiving multicast video
```

Comment:
Well, if you have a mountpoint bind to port 5002, and you have multiple interfaces, currently the socket will bind to them all. We should allow binding only to, e.g., eth0, and not eth1 too, for increased protection and control. This is what we already provide for the HTTP and WebSockets servers for instance.